modules/user/user.graphql.js

```diff
@@ -23,7 +23,7 @@ type User{
 
 const userInput = `
 input UserInput {
-    _id:ID!
+    _id:ID
```

Comment:
I think we should not have the `_id` field here at all, on creation client cannot provide the of any user because it is generated on the backend, on updating client doesn't have the ability to update the `_id`

Reply:
solved